.kokoro

```diff
@@ -271,6 +271,12 @@ run_bazel() {
   selected_xcode_developer_path=$(xcode-select -p)
   selected_xcode_contents_path=$(dirname "$selected_xcode_developer_path")
 
+  # todo fix error about missing plist
+  # 
+  # Running bazel builds...
+  # cat: /Library/Developer/version.plist: No such file or directory
+  #
+  # fixed by choosing command line tool from xcode.
```

Comment:
what do you mean by choosing command line tool from xcode?

Reply:
https://www.embarcadero.com/starthere/xe5/mobdevsetup/ios/en/installing_the_commandline_tools.html

Reply:
Is this different from `xcode-select`?

Reply:
probably not different from xcode-select